backend/Persistence/Repositories/MetadataCategoryRepository.cs

```diff
@@ -37,6 +37,7 @@ public async Task<IEnumerable<MetadataCategory>> ListRootElementsAsync()
 
         public async Task AddAsync(MetadataCategory category) {
             await _context.MetadataCategory.AddAsync(category);
+            await _context.SaveChangesAsync();
```

Comment:
It doesn't work without this? I thought you only needed SaveChangesAsync when you modified entries

Reply:
Hmmm, it may work without this. I will spin up Docker and check.

Reply:
Fixed now, did not need it.